packages/central-server/src/apiV2/changesMetadata.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { respond } from '@tupaia/utils';
+import { getPermissionsBasedChangesFilter } from './utilities';
+import { allowNoPermissions } from '../permissions';
+
+/**
+ * Permissions based sync metadata
+ * {
+ *   changeCount: number of changes since last sync
+ *   countries: countries included in the sync
+ *   permissionGroups: permissions groups included in the sync
+ * }
+ * Responds to GET requests to the /changes/metadata endpoint
+ */
+export async function changesMetadata(req, res) {
```

Comment:
Given that this endpoint returns more/different data to the `/changes/count` endpoint, I decided to create a new one with a new name. Also makes maintaining support for the old logic a little easier.